### PR TITLE
Implement insert_records_to_existing_table 

### LINKED ISSUE #1488: Implement insert_from_select to allow the inserting of records into an existing table.
## Description

To import records from the temp table to the target/existing table we will implement the `insert_from_select()` method in the db app and the `insert_records_to_existing_table()` method that the `/existing_import` endpoint would call.

## Expected behavior

Making a POST request to the `/existing_import` endpoint should call `insert_records_to_existing_table()` which will then call the `insert_from_select()` method and the records from the temp table will get inserted into the target table. 

For this issue, we are not worrying about the column mappings it will be dealt with in the upcoming tickets.

### PR DESCRIPTION
**Pull request by Anish9901**
Fixes #1488

This PR adds the `insert_from_select()` functionality for the tables for inserting records from the temp table to the target/existing table.

**Technical details**

```
POST /api/db/v0/tables/<temp_table_id>/existing_import/
```
**Request**
```
{
    "import_target": <target_table_id>,
    "data_files": [<data_file_id>],
    "mappings": null
}
```
**Screenshots**

## Before:

Screenshot from 2022-08-01 02-44-58

Screenshot from 2022-08-01 02-46-29

## After:

Screenshot from 2022-08-01 02-48-19

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: db/tables/operations/insert.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    target_table_col_list = [col for col in target_table.c if col.name != 'id']
+    with engine.begin() as conn:
+        sel = select([col for col in from_table.c if col.name != 'id'])
+        ins = target_table.insert().from_select(target_table_col_list, sel)
+        try:
+            result = conn.execute(ins)
+        except Exception as e:
+            # ToDo raise specific exceptions
+            raise e
+    return target_table, result
```
Comment (mathemancer) [inline db/tables/operations/insert.py:14]: I think this would be better in the `db.records.operations.insert` namespace.

**Review by mathemancer — CHANGES_REQUESTED** [review]
Overall, looks good. I like the change to `target_table`. The only request I have is to put the `insert_from_select` logic under the `db.records.operations.insert` namespace.

File: db/records/operations/insert.py [hunk 2]
```diff
@@ -72,3 +73,16 @@ def insert_records_from_csv(table, engine, csv_filepath, column_names, header, d
                         temp_file.write(contents)
                     temp_file.seek(0)
                     cursor.copy_expert(copy_sql, temp_file)
+
+
+def insert_from_select(from_table, target_table, engine, mappings=None):
+    target_table_col_list = [col for col in target_table.c if col.name != 'id']
+    with engine.begin() as conn:
+        sel = select([col for col in from_table.c if col.name != 'id'])
```
Comment (Anish9901) [inline db/records/operations/insert.py:81]: @mathemancer Is there any other way we can think of not considering the "id" column for the select statements?

File: db/records/operations/insert.py [hunk 3]
```diff
@@ -72,3 +73,16 @@ def insert_records_from_csv(table, engine, csv_filepath, column_names, header, d
                         temp_file.write(contents)
                     temp_file.seek(0)
                     cursor.copy_expert(copy_sql, temp_file)
+
+
+def insert_from_select(from_table, target_table, engine, mappings=None):
+    target_table_col_list = [col for col in target_table.c if col.name != 'id']
```
Comment (mathemancer) [inline db/records/operations/insert.py:79]:
```suggestion
    target_table_col_list = [
        col for col in target_table.c if not MathesarColumn.from_column(col).is_default
    ]
```

File: db/records/operations/insert.py (hunk 2 above)
Comment (mathemancer) [reply db/records/operations/insert.py:81]: Good idea! See my above suggestion (obviously, the relevant import needs to be made).

**Review by mathemancer — CHANGES_REQUESTED** [review]
Sorry to go for another review round, but your question was a good one, and it would be better to change places where you're using the `id` name to identify default columns as per my suggestion.

**Review by mathemancer — APPROVED** [review]
Alright, this looks good to me.

Thank you!

**Pull request merged**